Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -248,5 +243,27 @@ public static class WebApiServiceCollectionExtensions
 
             return services;
         }
+
+        internal static List<string> GetValidAudiences(
+            JwtBearerOptions options,
+            MicrosoftIdentityOptions msIdentityOptions)
+        {
+            var validAudiences = new List<string>();
+
+            // The valid audiences are both the Client ID (options.Audience) and api://{ClientID}
+            // If the developer doesn't set the Audience on JwtBearerOptions, use ClientId from MicrosoftIdentityOptions
+            if (!string.IsNullOrEmpty(options.Audience))
+            {
+                validAudiences.Add(options.Audience);
+                validAudiences.Add($"api://{options.Audience}");
```

Comment:
Do we want to have this one? if the developer does set an audience, we should just use it?